databases/week2/Qestion No.1.txt

```diff
@@ -0,0 +1,29 @@
+/*Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id*/
+
+	CREATE TABLE `task`(
+	`id` int unsigned auto_increment,
+	`title` varchar(255) not Null,
+	`description` text,
+	`created`datetime not Null,
+	`updated` datetime not Null,
+	`due_date` datetime not Null,
+	`status_id` int unsigned ,
+ 	`user_id` int unsigned  not Null,
+ 	primary key (`id`)
+ 	) ENGINE=INNODB  Default CHARSET=utf8mb4 collate=utf8mb4_unicode_ci;
+
+
+/* Change the title of a task*/
+	ALTER TABLE task 
+	CHANGE  title Task_title VARCHAR(255) ;
+
+/*Change a task due date*/
+	ALTER TABLE task 
+   	CHANGE  due_date D_date  datetime not null;
+
+/* Change a task status*/
+	ALTER TABLE task 
+	CHANGE  status_id statusID  int unsigned not null;
+
+/*Mark a task as complete*/
+	DROP TABLE task;
```

Comment:
I think you misunderstood this part of the exercises. The idea is to insert, update and delete tasks, not to create new tables or modify columns